gradle.properties

```diff
@@ -0,0 +1,2 @@
+groovyVersion=2.5.7
```

Comment:
Why commit this instead of using the property to pass it?

Reply:
From the experience of maintaining `ci-health` project: https://github.com/gradle/ci-health/blob/cff9a8ba96e16114d47714443c570ac73fb259f1/gradle.properties#L4 I think putting the version into properties file is a good practice: I can easily know what's the latest version by examining this file and you still have the flexibility.

Reply:
Alright, you convinced me. :) I guess if we don't load the version number from Git then it's not such a big issue...